Validation error for `clock_gettime`

**Describe the bug**
There's a type-error when building code that refers to `clock_gettime`. Note that it's not calling `clock_gettime` that fails but rather the compilation. I don't expect asterius to implement `clock_gettime`.

**To Reproduce**
```haskell
module Main(main) where

import System.Clock

main :: IO ()
main = print =<< getTime Monotonic
```
Run with: `ahc-link --input-hs test.hs`

Error message:
```
root@45e1ebcefc2f:/workspace# ahc-link --input-hs doc_parA.hs
[INFO] Compiling doc_parA.hs to WebAssembly
[INFO] Converting linked IR to binaryen IR
[INFO] Running binaryen optimization
[INFO] Validating binaryen IR
[wasm-validator error in function clockzm0zi8zmEufwPgpWNwb1blvau8N0t6_SystemziClock_.Lchuc] none != i64: call type must match callee return type, on 
[none] (call $clock_gettime
 [i64] (i64.load align=1
  [i32] (i32.wrap_i64
   [i64] (i64.add
    [i64] (i64.load align=1
     [i32] (i32.const 1880)
    )
    [i64] (i64.const 8)
   )
  )
 )
 [i64] (local.tee $0
  [i64] (i64.load align=1
   [i32] (i32.wrap_i64
    [i64] (i64.add
     [i64] (i64.load align=1
      [i32] (i32.const 1048)
     )
     [i64] (i64.const 7)
    )
   )
  )
 )
)
ahc-link: user error ([ERROR] binaryen validation failed)
```

**Expected behavior**
I had hoped the code would compile (but not run correctly if `getTime` is on a hot code path).

**Environment**
 - OS name + version: Docker
 - Version of the code: latest

--- a/asterius/src/Asterius/CodeGen/Droppable.hs
+++ b/asterius/src/Asterius/CodeGen/Droppable.hs
@@ -8,5 +8,6 @@ where
 import Asterius.Types
 
 ccallResultDroppable :: EntitySymbol -> [ValueType]
-ccallResultDroppable sym | sym == "memset" || sym == "memcpy" = [I64]
+ccallResultDroppable sym
+  | sym `elem` ["memset", "memcpy", "clock_gettime", "clock_getres"] = [I64]
 ccallResultDroppable _ = []

